azure-pipelines/run_test_steps.yml

```diff
@@ -56,5 +56,5 @@ steps:
     env:
       TWISTED_REACTOR: ${{ parameters.windowsReactor }}
 
-- script: 'python -m tox -e coverage-prepare,coveralls-push'
+- script: 'python -m tox -e coverage-prepare,codecov-push,coveralls-push'
```

Comment:
In the future please put changes like this into a separate PR so we can merge them independently.